cmd/dockerregistry/main.go

```diff
@@ -16,6 +18,18 @@ import (
 )
 
 func main() {
+	// Start the profiling server, if requested.
+	profilingAddr := os.Getenv("REGISTRY_PROFILING_ADDR")
```

Comment:
@smarterclayton 
>Use the same env we use on Openshift binaries, both address and the web bit. In fact, just use the serviceability package.

I was just looking through the serviceability package, and all it provides is a deferred shutdown hook which writes a single kind of profiler output to a file on program exit. That's not equivalent to what I'm providing in this PR (a live net/pprof session over HTTP). Am I missing something?